worker/worker.go

```diff
@@ -342,6 +343,17 @@ func (w *worker) fetchContracts(ctx context.Context, metadatas []api.ContractMet
 	return
 }
 
+func (w *worker) fallbackAddr(ctx context.Context, hk types.PublicKey) string {
+	host, err := w.bus.Host(ctx, hk)
+	if err != nil {
+		return ""
+	}
+	if len(host.Subnets) > 0 {
```

Comment:
We can unfortunately not reuse `Subnets`. We are no longer returning subnets so we should add a new `ResolvedAddresses` field.
For compatibility reasons, the store can still populate the `Subnets` field by deriving them from `ResolvedAddresses` after reading the addresses from the database.